src/main/groovy/org/shipkit/internal/gradle/GitPlugin.java

```diff
@@ -154,6 +142,25 @@ public void execute(final Exec t) {
         });
     }
 
+    private List<String> getAddCommand(List<String> files) {
```

Comment:
Nitpick (feel free to ignore ;). Using "asList()" gives more compact code that could be a tiny bit more readable.